performance_manager/lib/rt_vehicle_positions.py

```diff
@@ -70,6 +75,98 @@ def transform_vp_dtyes(vehicle_positions: pandas.DataFrame) -> pandas.DataFrame:
         .astype("Int64")
     )
 
+    return vehicle_positions
+
+
+def join_gtfs_static(
+    vehicle_positions: pandas.DataFrame, db_manager: DatabaseManager
+) -> pandas.DataFrame:
+    """
+    join vehicle position dataframe to gtfs static records
+
+    adds "fk_static_timestamp" and "parent_station" columns to dataframe
+    """
```

Comment:
function is very close to trip update function, could potentially be combined in utils file